edi_voxel_stock_picking_oca/models/res_partner.py

```diff
@@ -0,0 +1,14 @@
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    voxel_report_id = fields.Many2one(
```

Comment:
This name is a lot generic...
Whats happend if I want a specific report for voxel xml report invoices??
So I think that the name could be voxel_picking_report_id...